src/migrations/1737745803297_add-blocks.ts

```diff
@@ -0,0 +1,35 @@
+import { MigrationBuilder, ColumnDefinitions, PgType } from 'node-pg-migrate'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('blocks', {
+    id: {
+      type: PgType.UUID,
+      primaryKey: true
+    },
+    blocker_address: {
+      type: PgType.VARCHAR,
+      notNull: true
+    },
+    blocked_address: {
+      type: PgType.VARCHAR,
+      notNull: true
+    },
+    blocked_at: {
```

Comment:
do we need to sort/paginate blocked users as well? shall we add a new index for blocked_at to make that sort/pagination faster or do you think it is not needed?